src/app/Traits/ExistingUserUniqueIdentifier.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace NetworkRailBusinessSystems\UserLogin\Traits;
+
+trait ExistingUserUniqueIdentifier
```

Comment:
This is very hard to read, and overcomplicated.

Just provide one trait for LDAP, and one trait for DB.

Reply:
Done, added separate traits for both 👍